bridge/tslint.json

```diff
@@ -89,7 +89,7 @@
     ],
     "trailing-comma": false,
     "no-conflicting-lifecycle": true,
-    "no-host-metadata-property": true,
+    "no-host-metadata-property": false,
```

Comment:
instead of disabling this rule globally, you could just add the tslint disable comment for the single line where it's needed

Reply:
I taked with Ermin about this ... we have this host metadata for our components on several files. I researched a bit and it totally valid to use this instead of @HostListener decorator in angular. We want to continue to use this as it is and therefore it makes sense to disable the rule in tslint.